grammar/fbp.peg

```diff
@@ -267,6 +353,7 @@ line
   / _ "OUTPORT=" node:node "." port:portName ":" pub:portName _ LineTerminator? {return parser.registerOutports(node,port,pub)}
   / _ "DEFAULT_INPORT=" name:portName _ LineTerminator? { defaultInPort = name}
   / _ "DEFAULT_OUTPORT=" name:portName _ LineTerminator? { defaultOutPort = name}
+  / annotation:annotation [\n\r\u2028\u2029] { return parser.registerAnnotation(annotation[0], annotation[1]); }
```

Comment:
An aside, using a named rule `newline` for `[\n\r\u2028\u2029]` would improve our error messages a bit

Reply:
Yep, let me tweak